src/bindings/c/include/openvino/c/ov_node.h

```diff
@@ -17,12 +17,21 @@
 typedef struct ov_output_const_node ov_output_const_node_t;
 typedef struct ov_output_node ov_output_node_t;
 
+/**
+ * @struct ov_output_const_node_list_t
+ * @brief Represents an array of const ov_output_nodes.
+ */
+typedef struct {
+    ov_output_const_node_t* output_ports;
```

Comment:
Maybe we can use more short name of variable, for example `ports` or `arr`.

Reply:
ports is ok

Reply:
Update it in new commit